src/util/exportToNDJson.js

```diff
@@ -59,12 +87,20 @@ const exportToNDJson = async (clientId, types) => {
  * @param {string} collectionName The collection of interest in the mongodb
  * @returns {Object} An object containing all data from the given collection name as well as the collection name
  */
-const getDocuments = async (db, collectionName) => {
+const getDocuments = async (db, collectionName, typefilterLookup) => {
   const query = {};
```

Comment:
This is a bit of a nitpick, but I think it might be clearer to update the query const (make it a `let` or similar) with `processTypeFilter(typefilterLookup[collectionName.toString()]` (using the appropriate rules from the else case below) and then use that query object in a single `db.collection(collectionName.toString()).find(query ...` especially since the query const is just an empty `{}`